packages/indexer/src/sinks/drizzle/transaction.ts

```diff
@@ -95,7 +78,7 @@ export class DrizzleSinkTransaction<
         return {
           ...originalFrom,
           where: (where: SQL) => {
-            // TODO
+            // TODO: confirmation needed
             const newWhere = sql`${where} AND upper_inf(${sql.raw("_cursor")})`;
```

Comment:
What if somehow the where clause is empty? Is that even possible? I'm worried the resulting sql will be invalid (`AND upper_inf(...)`).

Reply:
> What if somehow the where clause is empty? Is that even possible? I'm worried the resulting sql will be invalid (`AND upper_inf(...)`).

Good catch, i will fix this.